africanus/calibration/tests/test_phase_only.py

```diff
@@ -107,9 +20,15 @@ def test_phase_only_diag_diag():
     np.random.seed(42)
```

Comment:
There's more of explanation for this suggestion lower down.

```suggestion
    rs = np.RandomState(42)
```